commitlint.config.ts

```diff
@@ -300,12 +300,20 @@ module.exports = {
                     let message = `Please do not exceed ${maxLineLength} characters in title (found ${headerLength}).`;
                     if (!headerStr.startsWith('Merge ') && headerLength > maxLineLength) {
                         offence = true;
+
+                        let colonIndex = headerStr.indexOf(':');
+
+                        let titleWithoutArea = headerStr;
+                        if (colonIndex > 0){
+                            titleWithoutArea = headerStr.substring(colonIndex);
+                        }
+                        
                         let numRecomendations = 0;
-                        let lowerCaseHeaderStr = headerStr.toLowerCase()
+                        let lowerCaseTitleWithoutArea = titleWithoutArea.toLowerCase();
                         Object.entries(abbr).forEach(([key, value]) => {  
-                            if (lowerCaseHeaderStr.includes(key.toString())){
+                            if (lowerCaseTitleWithoutArea.includes(key.toString())){
                                 if (numRecomendations === 0) {
-                                    message = message + ' The following replacement(s) in your commit title are recommended:\n'
+                                    message = message + ' The following replacement(s) in your commit title are recommended:\n';
```

Comment:
@realmarv this line doesn't need to change in this PR (I know it's missing a semicolon, but it works; and if I let you change it, then I would be ok by making you arbitrary style changes in any other part of the file(s), which is not acceptable for a non-refactoring PR); sorry to be picky, but the point of this is to prevent git-blame noise; and by using Prettier soon (thanks to your other PR), we will avoid these problems later by fixing the root cause

Reply:
Right. Done